tests/fixtures/make_addons/full/config/copilot/web/manifest.yml

```diff
@@ -0,0 +1,151 @@
+  PERM_SEC_NAME: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/PERM_SEC_NAME
+  PERSON_UPDATE_HAWK_ID: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/PERSON_UPDATE_HAWK_ID
+  PERSON_UPDATE_HAWK_KEY: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/PERSON_UPDATE_HAWK_KEY
+  PERSON_UPDATE_WEBHOOK_URL: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/PERSON_UPDATE_WEBHOOK_URL
+  PFM_AWS_ACCESS_KEY_ID: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/PFM_AWS_ACCESS_KEY_ID
+  PFM_AWS_SECRET_ACCESS_KEY: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/PFM_AWS_SECRET_ACCESS_KEY
+  PFM_AWS_STORAGE_BUCKET_NAME: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/PFM_AWS_STORAGE_BUCKET_NAME
+  PFM_REGION: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/PFM_REGION
+  PROFILE_DELETED_EMAIL_TEMPLATE_ID: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/PROFILE_DELETED_EMAIL_TEMPLATE_ID
+  PROFILE_DELETION_REQUEST_EMAIL: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/PROFILE_DELETION_REQUEST_EMAIL
+  PROFILE_DELETION_REQUEST_EMAIL_TEMPLATE_ID: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/PROFILE_DELETION_REQUEST_EMAIL_TEMPLATE_ID
+  PROFILE_EDITED_EMAIL_TEMPLATE_ID: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/PROFILE_EDITED_EMAIL_TEMPLATE_ID
+  SENTRY_AUTH_TOKEN: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/SENTRY_AUTH_TOKEN
+  SENTRY_DSN: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/SENTRY_DSN
+  SKIP_LIST: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/SKIP_LIST
+  SUPPORT_REQUEST_EMAIL: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/SUPPORT_REQUEST_EMAIL
+  TWITTER_ACCESS_SECRET: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/TWITTER_ACCESS_SECRET
+  TWITTER_ACCESS_TOKEN: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/TWITTER_ACCESS_TOKEN
+  TWITTER_DEPT_USER: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/TWITTER_DEPT_USER
+  TWITTER_OAUTH_CONSUMER_KEY: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/TWITTER_OAUTH_CONSUMER_KEY
+  TWITTER_OAUTH_CONSUMER_SECRET: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/TWITTER_OAUTH_CONSUMER_SECRET
+  WAGTAIL_BASE_URL: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/WAGTAIL_BASE_URL
+  X_AUTOSCALING_MAX: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/X_AUTOSCALING_MAX
+
+
+# You can override any of the values defined above by environment.
+environments:
```

Comment:
I think we can also remove some of the environments as well... maybe one prod. environment and one non-prod. environment.

Reply:
These last two I'm inclined to leave as is. If the test fails due to a change in how/what addon config is generated, then leaving it as is means the expected data is easy to pull back in from the copliot check-cloudformation command.

Reply:
The environments and secrets/env vars are taken from the `bootstrap.yml` file, so if we strip them from there, then the generation of these files will be a lot less than what we have now. It also makes our code base a bit more secure as we don't publicise potential env vars that are used within DBT applications.

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/138/commits/fa584a8a39aafc481d523c696b86e40c0dec4a8e.